Conclusion (#22)
greenelab/knowledge-graph-review: content/03.text-mining.md

Overall unsupervised methods provide a means to rapidly extract relationship asserting sentences without the need of annotated text.
Approaches in this category range from calculating co-occurrence scores to clustering sentences and provide a generalizable framework that can be used on large repositories of text.

> Full text has already been show to make a dramatic improvements for methods that aim to infer relationships using cooccurrences [@doi:10.1371/journal.pcbi.1005962], and we should expect similar benefits for machine learning approaches.

Reviewer: ```suggestion
Full text has already been show to meaningfully improve the performance of methods that aim to infer relationships using cooccurrences [@doi:10.1371/journal.pcbi.1005962], and we should expect similar benefits for machine learning approaches.
```

Overall unsupervised methods provide a means to rapidly find relationship asserting sentences without the need of annotated text.
Approaches in this category range from using co-occurrence scores to clustering sentences.
These methods provide a generalizable framework that can be used on large repositories of text.

> Future methods can improve detection power by considering the use of methods that simplify sentences and use datasets that include full text articles.

Reviewer: ```suggestion
Future methods can improve detection power by considering the use of methods that simplify sentences and use datasets that include full text articles.
```
Do you want to cite the paper that talks about cooccurrences improving with full text? Something like:
> Full text has already been show to make a dramatic improvements for methods that aim to infer relationships using cooccurrences[ref], and we should expect similar benefits for machine learning approaches.
